applications/RANSApplication/tests/ChannelFlowTest/periodic_k_epsilon_channel_flow.mdpa

```diff
@@ -0,0 +1,11516 @@
+Begin ModelPartData
```

Comment:
This file is quite big, I would consider moving it to the Examples directory.

Reply:
done :)